11/1/2018 8:00:08 PM
Action failed!  The program encountered an exception writing to the log file.
The process cannot access the file 'C:\Workspace\team\week-4-pair-exercises-team-1\c#-capstone\etc\Log.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append)
   at Capstone.Log.WriteLog(String logDesc, Decimal logStartBalance, Decimal logEndBalance) in C:\Workspace\team\week-4-pair-exercises-team-1\c#-capstone\VendingMachine\Log.cs:line 31

